.gitignore

```diff
@@ -16,3 +16,5 @@
 /notebooks*/
 /rsconnect_jupyter/static/version.json
 /rsconnect_jupyter/version.py
+/cypress/screenshots/*
```

Comment:
capturing these artifacts during a test failure is :rocket: 

something like: https://github.com/rstudio/connect-setup/blob/bc38465eb400ef1466f0f76428a0fed32715c1dd/.github/workflows/smoke-tests.yml#L58-L70